Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: consul
-version: 0.24.1
+version: 0.25.0
```

Comment:
Could you revert to the current version of the chart? We usually update it ourselves when we release a new version.

Reply:
@ishustava Revert done.